src/webapp/ldap_data.py

```diff
@@ -8,24 +8,24 @@
 
 def get_contact_cilogon_id_map(global_data):
     """ return contacts dict, limited to users with a CILogonID """
-    contacts = global_data.get_contacts_data().users_by_id;
+    contacts = global_data.get_contacts_data().users_by_id
     return { k: v for k, v in contacts.items() if v.cilogon_id is not None }
 
 
 # cilogon ldap query constants
 #_ldap_url = "ldaps://ldap.cilogon.org"
 #_username = "uid=readonly_user,ou=system,o=OSG,o=CO,dc=cilogon,dc=org"
-_basedn   = "o=OSG,o=CO,dc=cilogon,dc=org"
+_cilogon_basedn   = "o=OSG,o=CO,dc=cilogon,dc=org"
 
 
-def get_cilogon_ldap_id_map(ldap_url, username, ldappass):
+def get_cilogon_ldap_id_map(ldap_url, ldap_user, ldap_pass):
     """ return dict of cilogon ldap data for each CILogonID, with the
         structure: {CILogonID: { "dn": dn, "data": data }, ...} """
     server = ldap3.Server(ldap_url)
-    conn = ldap3.Connection(server, username, ldappass)
+    conn = ldap3.Connection(server, ldap_user, ldap_pass)
```

Comment:
> Some naming tweaks for consistency

I think my naming convention consistency consistently consists of being consistent with my mood at the time of writing ;)

Reply:
That's consistent with what I know about you ;)